pro.md

```diff
@@ -72,3 +72,22 @@ button.setOnClickListener(new View.OnClickListener() {
 </figure>
 
 The PIN value will never leave the device except in the USSD session. If the action has no PIN, but you want to skip the confirm screen you can pass any string value for the pin: `.extra("pin", "1234")`.
+
+#### PRO customization
+With the Pro SDK, you have access to pass in your own layout customization as a parameter into the sdk, starting from version v1.7.0
+
+To pass in your custom layout, use
+<figure>
+<pre><code class="java" data-lang="java">
+...
+builder.styleView(R.layout.yourlayoutname)
```

Comment:
Specify which builder with a line like `HoverParameters.Builder builder = new HoverParameters.Builder(context);` before the ...